test/utils/helpers.go

```diff
@@ -434,6 +504,28 @@ func CollectAgentInfo(kubctl *KubeCtl, dir string) error {
 	return nil
 }
 
+// CollectVMAgentLog collects VM agent log from all imported VMs.
+func CollectVMAgentLog(cloudVPC CloudVPC, dir string) error {
+	err := os.MkdirAll(dir, 0777)
+	if err != nil {
+		return err
+	}
+	vmCmd := []string{"cat", "/var/log/antrea/antrea-agent.log"}
+	for _, vm := range cloudVPC.GetVMs() {
+		output, err := cloudVPC.VMCmd(vm, vmCmd, 10*time.Second)
+		if err != nil {
+			continue
+		}
+		fn := path.Join(dir, vm+"-agent.log")
```

Comment:
Are you writing antrea-agent logs from all the VMs into one file?

Reply:
No, its vmID-agent.log